### PR TITLE
SSVC-318 Adding Master Data to Security Api Docs

### PR DESCRIPTION
**Pull request by binghamtj**
@ejcarr @brianhyder  @stuisme @brentpabst

### CODE REVIEW

File: apiary.apib [hunk 1]
```diff
@@ -64,17 +64,174 @@ Any response that does not provide a ```200 OK``` response code should be treate
...
 
 ### Properties
-+ `vitalCheck`: true (boolean, required) - Vital check to ensure something is online.
\ No newline at end of file
++ `vitalCheck`: true (boolean, required) - Vital check to ensure something is online.
+
+## UserDetail (object)
+Represents a UserDetail object
+
+### Properties
```
Comment (elizabethmidgett) [inline apiary.apib:122]: Are we switching to use the Properties notations on objects? (Just confirming because it does not exist in EventApi)

Comment (binghamtj) [inline apiary.apib:122]: I was just copying what I saw in the File API, so I don't know.

Comment (elizabethmidgett) [inline apiary.apib:122]: @brianhyder could you weigh in here :)

Comment (brianhyder) [inline apiary.apib:122]: This is the proper format when the object contains a description directly underneath it.  The explicit notation is fine.

**Pull request merged**